defaults/main/main.yml

```diff
@@ -0,0 +1,62 @@
+---
+nginx_user: "{% if ansible_os_family == 'RedHat' %}nginx{% elif ansible_os_family == 'Debian' %}www-data{% endif %}"
```

Comment:
Good trick!

Reply:
What happens if neither condition evaluates to true (e.g. not RedHat or Debian)?  Does the script fail, or just add a blank nginx user?  I know we only test RedHat and Debian but I think it would be good to define an explicit behaviour for other Linux flavours.

Reply:
Hi, it seems I have added it to role, but I only created a new main directory and seems I created the complete file.

This line is already included in master branch.

When `nginx_user` is not defined (because using another linux distro) the ansible task that uses the variable fails saying it is undefined.